modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-multiple-bsl-use-case_{context}"]
+= {oadp-short} use case for two BSLs
+
+In this use case, you configure the DPA with two storage locations by using two cloud credentials. You backup an application with a database by using the default BSL. {oadp-short} stores the backup resources in the default BSL. You then backup the application again by using the second BSL.
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must configure two backup storage locations, AWS S3 and Multicloud Object Gateway (MCG).
```

Comment:
I'm not sure what the intended meaning is here. As it is currently written, this sentence could be interpreted in two ways:
`You must configure two backup storage locations, AWS S3, and Multicloud Object Gateway (MCG).`
`You must configure two backup storage locations: AWS S3 and Multicloud Object Gateway (MCG).`

Reply:
It means the 2nd one. I have made the change. thanks.